doc/topics/cloud/vmware.rst

```diff
@@ -19,12 +19,11 @@ available at PyPI:
 
 https://pypi.org/project/pyvmomi/
 
-This package can be installed using `pip` or `easy_install`:
+This python module can be installed in the onedir package:
 
 .. code-block:: bash
 
-    pip install pyvmomi
-    easy_install pyvmomi
+    /opt/saltstack/salt/bin/pip3 install pyVmomi
```

Comment:
The ```/opt/saltstack/salt/bin/pip3``` would be better replaced with ```salt-pip``` since the path can change depending on the platform.

Reply:
indeed, thanks for the finding.